data/base.go

```diff
@@ -0,0 +1,286 @@
+package data
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	valid "github.com/tidepool-org/platform/Godeps/_workspace/src/gopkg.in/bluesuncorp/validator.v8"
+	"github.com/tidepool-org/platform/Godeps/_workspace/src/labix.org/v2/mgo/bson"
+
+	"github.com/tidepool-org/platform/validate"
+)
+
+//used for all data types
+var validator = validate.NewPlatformValidator()
+
+func init() {
+	validator.RegisterValidation(timeStrTag, TimeStringValidator)
```

Comment:
Ahhh, no abbreviations. :smile: `timeStringTag` and `timeObjectTag` (and their subsequent values).